modules/ROOT/partials/depl-examples/ubuntu-compose/shared-setup.adoc

```diff
@@ -0,0 +1,442 @@
+* `SMTP_SENDER` +
+An eMail address that is used for sending Infinite Scale notification eMails like +
+[.blue]##`ocis <\noreply@yourdomain.com>`##.
+* `SMTP_USERNAME` +
+Username for the SMTP host to connect to.
+* `SMTP_PASSWORD` +
+Password for the SMTP host to connect to.
+* `SMTP_AUTHENTICATION` +
+Authentication method for the SMTP communication.
+* `SMTP_SECURITY` +
+Define using secure or insecure connections to the SMTP server.
+--
+
+==== Domain Names
+
+This environment requires that multiple (sub)domain names are available. You have to deviate from the examples according your environment. The (sub)domains must be configured to point to the server you configured. The same ist true when using a wildcard configuration. For a wildcard setup on the DNS, three (sub)domains must be provided to configure the `.env` file described in one of the next sections.
+
+The following (sub)domains are required at minimum, an example is printed for each, replace them according your environment:
+
+* `OCIS_DOMAIN` +
+[.blue]##`ocis.yourdomain.com`##
+
+* `COLLABORA_DOMAIN` +
+[.blue]##`collabora.yourdomain.com`##
+
+* `WOPISERVER_DOMAIN` +
+[.blue]##`wopiserver.yourdomain.com`##
+
+== Accessing Infinite Scale
+
+Infinite Scale can be accessed from the internet and from your local network with the `OCIS_DOMAIN` you have defined:
+
+image::depl-examples/ubuntu-compose/{overview_image}[Network Overview, width=400]
+
+== Limitations
+
+Data Location::
+All data is stored in volumes managed by docker in the same partition of the server. You need to manually reconfigure the docker volumes in the main docker compose file if you want to define your own volume paths instead the ones managed by docker. This is highly recommended as described in the xref:hardware[Hardware] section above, but not part of this documentation.
+
+User and User Access Management::
+The following embedded services are well suited for home use and smaller businesses though Infinite Scale can be configured to use external products which is relevant for bigger installations and not covered here.
++
+--
+* Infinite Scale has an embedded identity management (IDM footnote:[See the xref:{s-path}/idm.adoc[IDM, window=_blank] service for more details]) which takes care of creating, storing, and managing user identity information.
+
+* In addition, it also has an embedded identity provider (IDP footnote:[See the xref:{s-path}/idp.adoc[IDP, window=_blank] service for more details]) to track and manage user identities, as well as the permissions and access levels associated with those identities.
+--
+
+end::shared_1[]
+
+
+tag::hetzner_only_1[]
+
+== Prepare Hetzner
+
+=== SSH Key
+
+To be prepared accessing the configured server, we recommend using key based authentication for ssh instead using user and password. This is not only beneficial for security reasons but also because you can define the public key to be installed  during the initial server configuration.
```

Comment:
```suggestion
We recommend using key based authentication for ssh to access the configured server instead of using user and password. This is not only beneficial for security reasons but also because you can define the public key to be installed  during the initial server configuration.
```